kemboijs/nodejs-api-cli
# #45 setup Postgres database without using ORM

#### What does this PR do?
- This PR allows  kemboijs-cli to set up Postgres database without ORM
- This PR also allows creation and dropping of the database without the need for ORM
 
#### Description of tasks to be completed.
- Added data  for deleting and creating a database and setting up a connection
#### How can this be manually tested?
- `git clone https://github.com/kemboijs/kemboijs-cli`
- `git checkout setup-postgres-without-orm`
-  `npm start init `
- Fill in the field  you are requested by the application (ensure you choose the no ORm option)
-  from your root directory, navigate to src/models/setUp.js
- You will find the setup for no orm option already in place.
